tests/drivers/build_all/regulator/i2c.dtsi

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/****************************************
+ * PLEASE KEEP REG ADDRESSES SEQUENTIAL *
+ ***************************************/
+
+pca9420@0 {
+	compatible = "nxp,pca9420";
+	reg = <0x0>;
+
+	BUCK1 {};
+	BUCK2 {};
+	LDO1 {};
+	LDO2 {};
+};
+
+npm6001@1 {
+	compatible = "nordic,npm6001";
+	reg = <0x1>;
+
+	regulators {
+		compatible = "nordic,npm6001-regulator";
+
+		BUCK0 {};
+		BUCK1 {};
+		BUCK2 {};
+		BUCK3 {};
+		LDO0 {};
+		LDO1 {};
```

Comment:
Sorry not related to this but... the uppercase here looks out of place, I could maybe understand it for LDO but buck is not an acronym, and it's not like we write stuff like SPI uppercase in the devicetree. Should these be lowercase for coherency with other devicetree stuff?

Reply:
It's following Linux convention for regulator node names

Reply:
Uh, can't argue with LINUX.

Reply:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/25011557/6558a83c-56c0-461c-89b5-3c553b67fd06)